tornado_stale_client/base.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of tornado-stale-client.
+# https://github.com/globocom/tornado-stale-client
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/MIT-license
+# Copyright (c) 2016, Globo.com <backstage@corp.globo.com>
+
+import json
+import logging
+from io import BytesIO
+from urllib.parse import urlencode
+
+import tornado
+from redis import StrictRedis
+from tornado import gen
+from tornado.httpclient import HTTPRequest, HTTPResponse
+from tornado.httputil import HTTPHeaders
+
+
+class BaseStaleHTTPClient(object):
```

Comment:
Can you keep only common methods in this base class? Looks like most methods here are overriden on the new client...